CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ This project adheres to [Semantic Versioning](http://semver.org).
 This document is formatted according to the principles of [Keep A CHANGELOG](http://keepachangelog.com).
 
 ## [Unreleased]
+### Changed
+- [Perl] Follow some Perl Best Practices (found by perlcritic)
```

Comment:
I assume there is no meaningful change for the end user. If so, this can be omitted.

Reply:
At this point, there's nothing for the end user, indeed.